### PR TITLE
Bugfix/director v2 loop

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
director-v2 startup script creates 2 event loops when in production mode.

Also fixes other fastapi-based applications (catalog, api-server, dynamic-sidecar).

### CODE REVIEW

File: services/api-server/src/simcore_service_api_server/main.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"""Main application to be deployed in for example uvcorn.
```
Comment (pcrespov) [inline services/api-server/src/simcore_service_api_server/main.py:1]: TYPO: unIcorn

**Review by pcrespov — APPROVED** [review]
Pair reviewed.

I wonder if we should then remove these fields from the ``AppSettings`` related to uvicorn since they are now 
```python
    # SERVICE SERVER (see : https://www.uvicorn.org/settings/)
    host: str = "0.0.0.0"  # nosec
    port: int = 8000
```
since now they are fix values in the ``docker/boot.sh``

File: services/api-server/src/simcore_service_api_server/main.py (hunk 1 above)
Comment (GitHK) [reply services/api-server/src/simcore_service_api_server/main.py:1]: someone had fun on Friday with pink unicorns 😉

**Review by GitHK — APPROVED** [review]
👍

**Comment by GitHK** [comment]
> Pair reviewed.
> 
> I wonder if we should then remove these fields from the `AppSettings` related to uvicorn since they are now
> 
> ```python
>     # SERVICE SERVER (see : https://www.uvicorn.org/settings/)
>     host: str = "0.0.0.0"  # nosec
>     port: int = 8000
> ```
> 
> since now they are fix values in the `docker/boot.sh`

If they are not used and it is not possible to overwrite them via env vars, I would remove them.

**Comment by pcrespov** [comment]
> > Pair reviewed.
> > I wonder if we should then remove these fields from the `AppSettings` related to uvicorn since they are now
> > ```python
> >     # SERVICE SERVER (see : https://www.uvicorn.org/settings/)
> >     host: str = "0.0.0.0"  # nosec
> >     port: int = 8000
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > since now they are fix values in the `docker/boot.sh`
> 
> If they are not used and it is not possible to overwrite them via env vars, I would remove them.

@GitHK What do you mean? All attributes of pydantic setting classes can be set via environment variables

**Comment by GitHK** [comment]
> > If they are not used and it is not possible to overwrite them via env vars, I would remove them.
> 
> @GitHK What do you mean? All attributes of pydantic setting classes can be set via environment variables

Yes that's what my understanding of the `BaseSettings` class is. Read [more here](https://pydantic-docs.helpmanual.io/usage/settings/). You can see in the example they state that if there is `env_prefix` all names will be fetched form the environment without using one. In this case `host`and `port`, if they are defined they will be fetched from there.

**Pull request merged**